starcluster/plugins/efs.py

```diff
@@ -29,14 +29,16 @@ class EFSPlugin(clustersetup.DefaultClusterSetup):
     SETUP_CLASS = starcluster.plugins.efs.EFSPlugin
     mount_point = /mnt/myefs
     fs_id = fs-1234abcd
+    mount_options = rsize=1048576, wsize=1048576, hard, timeo=600, retrans=2
```

Comment:
@cariaso you're correct. I forgot to remove the spaces in the example. The actual defaults do not contain the spaces. I've committed the change.